snooty/postprocess.py

```diff
@@ -210,12 +212,23 @@ def get_include_argument(node: n.Directive) -> str:
             assert len(argument_list) > 0
             return argument_list[0].value
 
-        if not isinstance(node, n.Directive) or not node.name == "include":
+        if not isinstance(node, n.Directive) or not node.name in [
+            "include",
+            "sharedinclude",
+        ]:
             return
 
         argument = get_include_argument(node)
         include_slug = clean_slug(argument)
+
+        # TODO PICKING UP ON TUESDAY: see notes in Notion
+
+        # if node.name == "sharedinclude":
+        #     filename: Path = Path(DEFAULT_CACHE_DIR, include_slug)
+        #     include_fileid: FileID = FileId(filename)
+        # else:
```

Comment:
I've not been able to deduce why `self.slug_fileid_mapping.get(include_slug)` isn't finding the shared include. My notes with copious debugging `print()` statements and their output are [here](https://gist.github.com/schmalliso/aff92f70f2898fb6031d91c9fcbd6905). 🆘